server/src/main/scala/org/http4s/server/middleware/authentication/DigestAuth.scala

```diff
@@ -77,6 +78,7 @@ object DigestAuth {
       }
     }
 
+  @nowarn("cat=other-match-analysis")
```

Comment:
2.13.4 wants a `case Some(Authorization(_))`, which exists?

Reply:
is `Authorization(_)` irrefutable?

Reply:
The warning goes away if `Authorization.unapply` is declared to return `Some` instead of `Option`, but that breaks binary compatibility.  The error is confusing, because it asks to repeat the case verbatim, but I can also see why scalac sees a leak.

With fresh eyes, I think `Some(_)` would be correct, be complete, and appease scalac.